spec/models/item_spec.rb

```diff
@@ -43,81 +52,60 @@
-      item_order8 = ItemOrder.create!(item_order8_info)
-
-      expect(Item.most_popular_items).to eq([@item_3.name, @item_1.name, @item_5.name, @item_7.name, @item_4.name])
-      
-      expect(Item.least_popular_items).to eq([@item_2.name, @item_6.name, @item_4.name, @item_7.name, @item_5.name])
-      
-      # order1 has items : item1, item3, item5, item7
-      # order2 has items : item2, item4, item6, item1
-      # total quantities ordered :
-      # item3 = 100
-      # item1 = 69
-      # item5 = 56
-      # item7 = 54
-      # item4 = 27
-      # item6 = 23
-      # item2 = 20
+    it "most and least popular items" do
+      merchant = Merchant.create!(name: "Meg's Bike Shop", address: '123 Bike Rd.', city: 'Denver', state: 'CO', zip: 80_203)
+      item_1 = merchant.items.create!(name: 'Gatorskins', description: "They'll never pop!", price: 100, image: 'https://www.rei.com/media/4e1f5b05-27ef-4267-bb9a-14e35935f218?size=784x588', inventory: 12)
+      item_2 = merchant.items.create!(name: 'Chain', description: "It'll never break!", price: 50, image: 'https://www.rei.com/media/b61d1379-ec0e-4760-9247-57ef971af0ad?size=784x588', inventory: 5)
+      item_3 = merchant.items.create!(name: 'Shimano Shifters', description: "It'll always shift!", price: 180, image: 'https://images-na.ssl-images-amazon.com/images/I/4142WWbN64L._SX466_.jpg', inventory: 2)
+      item_4 = merchant.items.create!(name: 'Boots', description: "Dont fear the rain!", price: 129, image: 'https://images-na.ssl-images-amazon.com/images/I/4142WWbN64L._SX466_.jpg', inventory: 6)
+      item_5 = merchant.items.create!(name: 'Bike Lights', description: "Dont fear the dark!", price: 62, image: 'https://images-na.ssl-images-amazon.com/images/I/4142WWbN64L._SX466_.jpg', inventory: 10)
+      item_6 = merchant.items.create!(name: 'Camelback water bottle', description: "Dont fear the thirst!", price: 42, image: 'https://images-na.ssl-images-amazon.com/images/I/4142WWbN64L._SX466_.jpg', inventory: 10)
+      item_7 = merchant.items.create!(name: 'Helmet', description: "Dont fear the risk!", price: 20, image: 'https://images-na.ssl-images-amazon.com/images/I/4142WWbN64L._SX466_.jpg', inventory: 15)
+
+      merchant_user_1 = User.create!(name: 'Mark',
+                                   street_address: '112 Elm St.',
+                                   city: 'Cambridge',
+                                   state: 'MA',
+                                   zip_code: "92139",
+                                   email: 'greek@examples.com',
+                                   password: 'hamburger03',
+                                   role: 1)
+
+      order_1 = @default_user_1.orders.create!(name: 'Bert', address: '123 Sesame St.', city: 'NYC', state: 'New York', zip: 10001)
+      order_2 = merchant_user_1.orders.create!(name: 'Mark', address: '112 Elm St.', city: 'Cambridge', state: 'MA', zip: 92139)
```

Comment:
I think this one would read better if a default_user placed the order(that being said, merchant employees are technically allowed to place orders too so it's possible that this would happen, just a bit more confusing I think).